recipe/meta.yaml

```
@@ -7,34 +7,50 @@ package:
 
 source:
   url: https://github.com/openforcefield/openff-toolkit/archive/{{ version }}.tar.gz
-  sha256: 2c94b79417d1217b15b68f40c09278d7c9e428e5527f928c8c29651ee07fbbb5
+  sha256: dc02048b46eb83abb7b1300cd979bc8ca52c8ad23c6f7de64a7af6d8135fe0da
 
 build:
-  noarch: python
-  number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
-
-requirements:
-  host:
-    - python
-    - pip
-  run:
-    - python >=3.6
-    - numpy
-    - smirnoff99frosst
-    - openff-forcefields
-    - openmm
-    - networkx
-    - rdkit
-    - ambertools >=20
-    - mdtraj
-    - xmltodict
-  run_constrained:
-    - openforcefield ==9999999999
-
-test:
-  imports:
-    - openff.toolkit
+  number: 1
+
+outputs:
+  - name: openff-toolkit-base
+    script: build_base.sh
+    build:
+      noarch: python
+
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+      run:
+        - python >=3.6
+        - numpy
+        - smirnoff99frosst
+        - openff-forcefields
+        - openmm
+        - networkx
+        - mdtraj
+        - xmltodict
+      run_constrained:
+        - openforcefield ==9999999999
+
+    test:
+      imports:
+        - openff.toolkit
+
+  - name: openff-toolkit
+    build:
+      noarch: python
+
+    requirements:
+      run:
+        - rdkit
+        - ambertools >=20
+        - {{ pin_subpackage('openff-toolkit-base', max_pin="x.x.x") }}
+
+    test:
+      imports:
+        - openff.toolkit
```

Notice that [no tests are being run for `openff-toolkit`](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=269305&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=1390). This might be due to the `pin_subpackage` issue above.

Also, add these to ensure we have the expected behavior:

```suggestion
        - openff.toolkit
        - rdkit
      commands:
        - python run_test.py
```